src/libopensc/card-cardos.c

```diff
@@ -1223,6 +1460,9 @@ cardos_card_ctl(sc_card_t *card, unsigned long cmd, void *ptr)
 	case SC_CARDCTL_CARDOS_GENERATE_KEY:
 		return cardos_generate_key(card,
 			(struct sc_cardctl_cardos_genkey_info *) ptr);
+	case  SC_CARDCTL_CARDOS_PASS_ALGO_FLAGS:
```

Comment:
clang-format suggested the following change:


```
-	case  SC_CARDCTL_CARDOS_PASS_ALGO_FLAGS:
+	case SC_CARDCTL_CARDOS_PASS_ALGO_FLAGS:
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->